### ISSUE TITLE
ft_inverse_sloreta is just ft_inverse_mne in disguise

### ISSUE THREAD

**Issue by schoffelen** [fieldtrip/fieldtrip#1483]
Since ft_inverse_sloreta is essentially a minnorm with a specific normalisation, it may be confusing to have 2 very similar implementations. ft_inverse_sloreta has been contributed by @sarangnemo , and is based on the formulation in the sekihara book. (The best part of) the ft_inverse_mne code is based on the formulation on the MNE website, and loosely based on how it's implemented in brainstorm. I think it would be good to stick to a single implementation, and consider moving the normalisation step to ft_sourcedescriptives (as it is currently for dSPM), or to implement a 'weightnorm' option, as per the beamforming functions.

**Comment by robertoostenveld** [comment]
That is a (valid) math-based argument. Citation-wise (e.g. it should ideally refer to the proper inital  https://www.uzh.ch/keyinst/NewLORETA/sLORETA/sLORETA-Math01.pdf paper, and on top of that it could also refer to the Sekihara book) and for people to find the method it would be better to keep it separate.

I think it is similar to the different beamforming implementations. The effort required to merge lcmv/pcc/dics (and possibly sam) would be much larger than "only" making the respective functions more consistent.  

Also, having relatively simple implementations will keep the overall code-base more accessible for others to contribute to. 

So in short: I don't see a big advantage.

**Comment by schoffelen** [comment]
OK, but then I suggest that the main stakeholders of the ft_inverse_sloreta code critically checks or more extensively documents it. Currently, it is using the lambda as a percentage in combination with a covariance matrix, while the lambda is used for inversion of the gram matrix. Also, implicitly, the provided covariance matrix is (I guess) supposed to be a noise covariance, not a data covariance, as per the beamformer functions, and I don't see any documentation about this. Also, ft_sourceanalysis does not seem to know this.

**Comment by robertoostenveld** [comment]
Feel free to add some `ft_warnings` to the code, referring to the book from which the equations were copied and the author of the code.

**Comment by schoffelen** [comment]
Closing, I have no incentive for improving the code in this corner of the codiverse
